components/sample-table.js

```diff
@@ -50,7 +50,7 @@ const sampleColumns = [
     id: "construct_library_sets",
     title: "Construct Library Set",
     display: ({ source }, { constructLibrarySetAccessions }) => {
-      if (source.construct_library_sets.length > 0) {
+      if (source.construct_library_sets && source.construct_library_sets.length > 0) {
```

Comment:
It would work to just use `source.construct_library_sets?.length > 0` just to save typing.